package.json

```
@@ -4,6 +4,11 @@
   "description": "Simple HTML5 charts using the canvas element.",
   "version": "3.5.1",
   "license": "MIT",
+  "type": "module",
+  "exports": {
+    "import": "./dist/chart.esm.js",
+    "require": "./dist/chart.js"
```

You'd also have to export `package.json`. Some build tools such as `rollup-plugin-svelte` will try to consume it. I'm not sure if this is exactly the right syntax, but I think you could do it with:
```suggestion
    "require": "./dist/chart.js",
    "./package.json": "./package.json"
```

---

I'm guessing `helpers` and `auto` need to be added too. And the `type` should not be changed, at least not before v4.